src/plot.cpp

```diff
@@ -1090,10 +1114,9 @@ void ProjectionPlot::set_output_path(pugi::xml_node node)
 // Advances to the next boundary from outside the geometry
 // Returns -1 if no intersection found, and the surface index
 // if an intersection was found.
-int ProjectionPlot::advance_to_boundary_from_void(GeometryState& p)
+int RayTracePlot::advance_to_boundary_from_void(GeometryState& p)
 {
-  constexpr double scoot = 1e-5;
-  double min_dist = {INFINITY};
+  double min_dist {INFINITY};
   auto coord = p.coord(0);
```

Comment:
Let's use a different variable name here so it doesn't shadow the `GeometryState` method.

Reply:
What method are you talking about?

```
gavin@Gavins-MBP openmc % grep -r "min_dist" include 
gavin@Gavins-MBP openmc % 
```

Reply:
Whoops! Comment was off by a line.

The variable `coord` on the line below is what I was referring to. It doesn't shadow _really_ (we'd know if it did), but I think a different name like `root_coord` would be more clear and reduce confusion with the `Particle::coord` method.